File: test/extra_test_output_test.py
import base64
import unittest

MSG = 'UmljY2FyZG8gbGlrZXMgcGluZWFwcGxlIHBpenphCg=='


class TestExtraTestOutput(unittest.TestCase):

    def test_extra_output(self):
        with open('truth.txt', 'wb') as f:
            f.write(base64.b64decode(MSG))

File: test/extra_test_output_test.go
package test

import (
	"encoding/base64"
	"os"
	"testing"
)

const msg = "UmljY2FyZG8gbGlrZXMgcGluZWFwcGxlIHBpenphCg=="

func TestWriteExtraOutput(t *testing.T) {
	out, err := base64.StdEncoding.DecodeString(msg)
	if err != nil {
		t.Errorf("Expected no error, got %s", err)
	}
	err = os.WriteFile("truth.txt", out, 0644)
	if err != nil {
		t.Errorf("Expected no error, got %s", err)
	}
}
